### PR TITLE
depth tests parameters

### PR DESCRIPTION
**Pull request by CedricGuillemet**
fixes https://github.com/BabylonJS/BabylonNative/issues/368
bjs update with commit https://github.com/BabylonJS/Babylon.js/pull/8967/commits/cab5feeee5a04ff74fb9d017502aa344f282bcf5

### CODE REVIEW

**Review by syntheticmagus — APPROVED** [review]

File: Plugins/NativeEngine/Source/NativeEngine.cpp [hunk 1]
```diff
@@ -1577,9 +1577,20 @@ namespace Babylon
 
         // TODO: support other fill modes
         uint64_t fillModeState = 0; //indexed tri list
-        if (fillMode == 2)
+        switch (fillMode)
```
Comment (bghgary) [inline Plugins/NativeEngine/Source/NativeEngine.cpp:1580]: I think we should try to be more consistent in passing enums. Sometimes we pass bgfx enums. Sometimes we pass Babylon enums. Should we just pick one and stick with it?

File: Plugins/NativeEngine/Source/NativeEngine.cpp [hunk 2]
```diff
@@ -1577,9 +1577,20 @@ namespace Babylon
...
         {
+        case 2: // MATERIAL_PointFillMode
             fillModeState |= BGFX_STATE_PT_POINTS;
```
Comment (bghgary) [inline Plugins/NativeEngine/Source/NativeEngine.cpp:1583]: Why are these `|=` and not just `=`?

**Review by bghgary — APPROVED** [review]

File: Plugins/NativeEngine/Source/NativeEngine.cpp (hunk 2 above)
Comment (CedricGuillemet) [reply Plugins/NativeEngine/Source/NativeEngine.cpp:1583]: Agree. Fixed.

File: Plugins/NativeEngine/Source/NativeEngine.cpp (hunk 1 above)
Comment (CedricGuillemet) [reply Plugins/NativeEngine/Source/NativeEngine.cpp:1580]: Agree. Opening an issue for that. https://github.com/BabylonJS/BabylonNative/issues/406

**Pull request merged**